lumberjack.go

```diff
@@ -105,11 +110,11 @@ type Logger struct {
 
 	// Compress determines if the rotated log files should be compressed
 	// using gzip. The default is not to perform compression.
-	Compress bool `json:"compress" yaml:"compress"`
-
-	size int64
-	file *os.File
-	mu   sync.Mutex
+	Compress    bool `json:"compress" yaml:"compress"`
+	createdTime int64
```

Comment:
Personally I prefer `createdAt` but up to you
```suggestion
	createdAt int64
```